src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -4457,6 +4464,165 @@ public unsafe bool IsReplicationCertificateByPublicKeyPinningHash(TransactionOpe
 
             return false;
         }
+
+        private unsafe void UpdateBackupHistory(ClusterOperationContext context, string type, long index, UpdatePeriodicBackupStatusCommand command,
```

Comment:
let's create a partial `ClusterStateMachine` and name it `ClusterStateMachine.Backup.cs` and move all backup related stuff there :)

Reply:
Same here: lost relevance due to changes.